CHANGES.rst

```diff
@@ -1,12 +1,31 @@
 Changes
 =======
 
-TBR
----
+0.13.0 (unreleased)
+-------------------
+
+* Cookie support is no longer experimental:
+
+  * If the ``COOKIES_ENABLED`` setting is ``True`` (default), automatic request
+    parameter mapping now sets ``responseCookies`` to ``True`` and maps request
+    cookies to ``requestCookies``.
```

Comment:
This means that, by default, *all* Zyte API requests with automap will start including `"responseCookies": True` among their parameters. Similar to how we do with `httpResponseHeaders`, but in this case the behavior also affects browser rendering and automatic extraction scenarios. The test expectation updates are a great way to get an idea of the impact.

This is a big one, the reason I went for 0.13.0 instead of 0.12.3, and I wonder whether or not this is the right call. I wonder if we should implement an opt-in setting for `responseCookies`, or make it so that it is only added to requests if `requestCookies` is also added (either manually by the user our automatically mapped).